packages/tocco-ui/src/Table/SortingState.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import {Icon} from '../'
+import {StyledSortingSpan} from './StyledTable'
+
+const SortingState = ({column}) => {
+  return column.sorting && column.sorting.sortRank
+    ? <StyledSortingSpan>
+      <Icon icon={`sort-${column.sorting.order === 'asc' ? 'up' : 'down'}`}/>
```

Comment:
`column.sorting` is not a required prop. accessing `column.sorting.order` without checking it first might lead to errors